core-bundle/src/Slug/Slug.php

```diff
@@ -27,7 +27,9 @@ public function __construct(private SlugGeneratorInterface $slugGenerator, priva
     }
 
     /**
-     * @param int|iterable $options A page ID, object or options array {@see SlugGeneratorInterface::generate()}
+     * @param int|iterable $options A page ID, object or options array
+     *
+     * @see SlugGeneratorInterface::generate()
```

Comment:
We must not use the inline `{@see …}` annotation, because it is apparently not supported by the phpdoc-parser and thus not associated with the `use` statements (see https://github.com/slevomat/coding-standard/issues/770).